### DISCUSSION TITLE
Omit Tables

### DISCUSSION THREAD

**Discussion by ksorbo** [spatie/laravel-backup#1429] [Ideas] [answered]
Is there a way to omit certain tables from the backup?

**Comment by erikn69** [comment, marked as answer]
yes, there is: `exclude_tables`
you only have to read documentation
there are many configuration options

**Reply by erikn69** [reply]
@ksorbo seriusly? Why do everyone find it so hard to read documentation? I'm not the autor, i only use search and comment here
https://spatie.be/docs/laravel-backup/v7/installation-and-setup

**Comment by ksorbo** [comment]
Can you send me a link to the page which shows this option? I cannot seem to find it. THANKS.

**Reply by ksorbo** [reply]
Thank you
